test/util/factories/profile_factory.dart

```diff
@@ -32,7 +32,10 @@ class ProfileFactory extends ModelFactory<Profile> {
       username: username ?? faker.internet.userName(),
       email: email ?? faker.internet.email(),
       description: getNullableParameterOr(description, faker.lorem.sentences(random.nextInt(2) + 1).join(' ')),
-      birthDate: getNullableParameterOr(birthDate, faker.date.dateTime()),
+      birthDate: getNullableParameterOr(
+          birthDate,
+          faker.date.dateTime(
+              minYear: DateTime(1900).year, maxYear: DateTime.now().subtract(const Duration(days: 365 * 12)).year)),
```

Comment:
nice, but please add a trailing comma
(and we should probably save the min age somewhere in the Profile class and only reference it here)